platform/lib/middleware/subdomain.js

```diff
@@ -53,6 +63,44 @@ class Subdomain {
       return next();
     };
   }
+
+  async redirectToReferrerOn404_(request, response) {
```

Comment:
That's really a nit, but I'd love this to have a short comment what problem this actually solves when we or somebody else is coming back to it later 🙂

Reply:
Makes sense. done.